hastexo/admin.py

```diff
@@ -101,13 +101,14 @@ class StackAdmin(admin.ModelAdmin):
 
     """
     form = StackAdminForm
-    list_display = ("name", "course_id", "status", "provider",
+    list_display = ("name", "course_id", student_email, "status", "provider",
                     "launch_timestamp",)
     list_filter = ("course_id", "status", "provider",)
     list_editable = ("status", "provider")
     list_select_related = True
     actions = [mark_suspended, mark_deleted, clear_stacklog]
-    search_fields = ("name", "course_id", "status", "provider")
+    search_fields = (
+        "name", "course_id", "learner__email", "status", "provider")
```

Comment:
Just checking, can you just outline why it's `student_email` with `list_display`, but `learner__email` with `search_fields`?

Reply:
Because displayable items have to be stack attributes or something callable, so we use the `student_email` function to display the email. Search fields however have to be actual model fields. If not, Django will throw errors :)

Reply:
Ah, gotcha. Thanks!